index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="container">
+        <nav>
+            <img class="logo" src="/image/adesoji-dara-logo.png" alt="logo-image">
+            <img class="menu" src="/image/hamburger-menu-icon.webp" alt="hambuger-icon">
+        </nav>
```

Comment:
You did a nice job using the semantic `<nav>` tag and kudos for using your logo! 💯 However, there's still room to improve:

- [ ] 🚧 The hamburger is having the wrong size and color, please keep them the same as what the FIGMA design suggests: `#6070FF` and `24px` wide. 🔧 

|Template|Your design|
|---|---|
| ![Screenshot from 2023-01-13 11-54-43](https://user-images.githubusercontent.com/77069011/212403993-f24189d6-1fc3-44d9-9fb5-38ae284785fe.png) | ![image](https://user-images.githubusercontent.com/77069011/212403399-3fbbc5d7-7d7e-440f-a35c-734cb6a91e81.png) |

- [ ] 🚧 By convention, the logo is a link that redirects the user to the home page, so in order to provide a good user experience, it would be good if you can wrap the `<img>` tag in an anchor `<a>` one. 🤓 🔧 

```suggestion
    <header>
        <nav>
            <a href="#">
                <img class="logo" src="/image/adesoji-dara-logo.png" alt="logo-image">
            </a>
            <img class="menu" src="/image/hamburger-menu-icon.webp" alt="hambuger-icon">
        </nav>
    </header>
```